scripts/configure-cluster/common/kubeconfigreset.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+reset_kubeconfig() {
```

Comment:
this can be in the kubeconfigandadmin.sh itself - setup and reset together in the same file

Reply:
Ideally yes, but since it exports envs, it needs to be called with . or source as in `. kubeconfigandadmin.sh`. This can result in the automatic execution of reset_kubeconfig as well. Hence why it is in a separate file.